plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoIcebergRestCatalogFactory.java

```diff
@@ -81,13 +83,18 @@ public synchronized TrinoCatalog create(ConnectorIdentity identity)
             warehouse.ifPresent(location -> properties.put(CatalogProperties.WAREHOUSE_LOCATION, location));
             properties.put("trino-version", trinoVersion);
             properties.putAll(securityProperties.get());
+
+            if (vendedCredentialsEnabled) {
+                properties.put("header.x-tabular-s3-access", "vended_credentials");
```

Comment:
We should probably standardize this to something like `header.x-iceberg-s3-access` so we don't reference Tabular specifically.  We can add this to the REST Spec as a signal that the client is requesting credentials.

Reply:
@danielcweeks was this ever standardized?